Project/src/MakeCall/RemoteParticipantCard.js

```diff
@@ -18,25 +19,42 @@ export default class RemoteParticipantCard extends React.Component {
         };
     }
 
-    componentWillMount() {
+    componentWillUnmount() {
+        this.remoteParticipant.off('isMutedChanged', () => {});
```

Comment:
Is this unsubscribing the correct listener?

Notice how we do this.remoteParticipant.on('isMutedChanged', () => {});

The () => {} is the listener.

The listeners are different objects in the on() and in the off(). So is this going to unsubscribe the correctly?

Reply:
It would be better to create the listener object and use that same reference